shuup/core/models/_categories.py

```diff
@@ -136,10 +136,10 @@ def is_visible(self, customer):
         return True
 
     @staticmethod
-    def _get_slug_name(self):
+    def _get_slug_name(self, translation):
```

Comment:
The reason we are passing translation object here is that the product/category-object might not contain the translation. Which will lead in the situation that the `safe_translation_getter` does not return right `title` for the object.

At least the `shuup.utils.multlanguage_model_form:_save_translations` causes situation that the object that we are generating slug for does not have translation even if the translation object is already in the database. Not sure if that is correct either but probably we don't want to change this functionality. @Pikkupomo what do you think about this?